.github/workflows/test_full.yml

```diff
@@ -10,15 +10,9 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ['3.7', '3.8', '3.9', '3.10', '3.11', '3.12']
-        django-version: ['<3.0', '<3.1', '<3.2', '<3.3', '<4.1', '<4.2', '<4.3']
+        python-version: ['3.8', '3.9', '3.10', '3.11', '3.12']
+        django-version: ['<3.3', '<4.2', '<4.3']
```

Comment:
I'm not really sure what's going on with `<` notation (which is my bad), but I don't I'm alone on it and I think it would be better if the version defined in a clear manner.